elasticsearch/src/main/java/org/hibernate/search/backend/elasticsearch/impl/ElasticsearchIndexManagerFactory.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.backend.elasticsearch.impl;
+
+import org.hibernate.search.engine.impl.DefaultIndexManagerFactory;
+import org.hibernate.search.indexes.spi.IndexManager;
+
+/**
+ * Index manager capable of resolving the "elasticsearch" alias. All other names/aliases will be handled by the default
+ * implementation.
+ * <p>
+ * Registered through the Java service loader mechanism.
+ *
+ * @author Gunnar Morling
+ */
+public class ElasticsearchIndexManagerFactory extends DefaultIndexManagerFactory {
```

Comment:
Oh, I see. I think for now I'll just add the "elasticsearch" name to the actual default factory then (yes, I know, knowledge of ES in core and all that... ;-)). But yes, let's generalize that scheme quickly.